src/formatted-codegen.js

```diff
@@ -542,14 +547,27 @@ export class ExtensibleCodeGen {
     return this.bracket(content, Sep.ARRAY_INITIAL, Sep.ARRAY_FINAL, Sep.ARRAY_EMPTY);
   }
 
-  reduceObjectAssignmentTarget(node, { properties }) {
-    let state = this.brace(this.commaSep(properties, Sep.OBJECT_BEFORE_COMMA, Sep.OBJECT_AFTER_COMMA), node, Sep.OBJECT_BRACE_INITIAL, Sep.OBJECT_BRACE_FINAL, Sep.OBJECT_EMPTY);
+  reduceObjectAssignmentTarget(node, { properties, rest }) {
+    let content;
```

Comment:
Is there a reason this implementation doesn't match that of `reduceObjectBinding`?

Reply:
It is exactly the same. Just the statement `content = this.commaSep(properties, Sep.OBJECT_BEFORE_COMMA, Sep.OBJECT_AFTER_COMMA);` is moved around. I will refactor to maintain consistency